.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        api-version: [5.5, 6.5]
+        api-version: ['5.5', '6.0', '6.5']
```

Comment:
Why not removing '5.5'?

Reply:
I just thought I had to do it in order
add 6.0 artifact (embedder) -> Change default version 5.5 to 6.0 (flutter-tizen) -> Remove 5.5 artifact(embedder)